scripts/release/mule/package/deb/package.sh

```diff
@@ -25,6 +24,7 @@ find tmp/node_pkgs -name "*${CHANNEL}*linux*${VERSION}*.tar.gz" | cut -d '/' -f3
     mkdir -p "${PKG_ROOT}/usr/bin"
     OS_TYPE=$(echo "${OS_ARCH}" | cut -d '/' -f1)
     ARCH=$(echo "${OS_ARCH}" | cut -d '/' -f2)
+    PKG_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH"
     OUTDIR="./tmp/node_pkgs/$OS_TYPE/$ARCH"
```

Comment:
This is the same as `OUTDIR`, so that could be repurposed to do double-duty.

Actually, if you could keep the name `PKG_DIR` rather than `OUTDIR` for both operations that would keep it in sync with other scripts that use that same variable name.